src/FluentAssertions.AspNetCore.Mvc/ActionResultAssertions.cs

```diff
@@ -145,7 +145,7 @@ public FileContentResultAssertions BeFileContentResult(string reason, params obj
         /// <summary>
         /// Asserts that the subject is an <see cref="FileStreamResult"/>.
         /// </summary>
-        internal FileStreamResultAssertions BeFileStreamResult()
```

Comment:
Wow. What a silly mistakes I made. Didn't realized. Thanks for the correction.

Reply:
No worries.